src/router/router.js

```diff
@@ -22,8 +22,10 @@ import { Log } from '../lib/log.js'
 import { stage } from '../launch.js'
 import Focus from '../focus.js'
 import Announcer from '../announcer/announcer.js'
+import { reactive } from '../lib/reactivity/reactive.js'
+import Settings from '../settings.js'
 
-export let currentRoute
```

Comment:
I don't think we can just remove this export. There may be apps that use this value that would then break because of this change, and I don't think we should do a major bump for this feature alone.

Can we add some code to make this backwards compatible? Possibly with a deprecation log when using `currentRoute` directly (if possible).